transactions/adminSetNFTInfo_FLOAT.cdc

```diff
@@ -0,0 +1,17 @@
+import Admin from "../contracts/Admin.cdc"
+import Dandy from 0x0afe396ebc8eee65
+
+transaction() {
+
+    let adminRef : &Admin.AdminProxy
+
+    prepare(account: AuthAccount){
+        self.adminRef = account.borrow<&Admin.AdminProxy>(from: Admin.AdminProxyStoragePath) ?? panic("Cannot borrow Admin Reference.")
+        
+    }
+    execute{
+        let type: Type = Type<@FLOAT.NFT>()
+        self.adminRef.setNFTInfo(alias: "FLOAT", type: type, icon: "https://testnet.floats.city/floatlogowebpage.png", providerPath: /private/floatHasNoPrivatePath, publicPath: FLOAT.FLOATCollectionPublicPath, storagePath: FLOAT.FLOATCollectionStoragePath, allowedFTTypes: nil, address: 0x0afe396ebc8eee65, externalFixedUrl: "testnet.floats.city")
```

Comment:
Instead of importing FLOAT here and having to have a lot of these third party contracts in our repo we do not need I think we should just copy the value of the paths over as text here, just set the private path as the same as public path.